src/Compilers/Core/Portable/Diagnostic/Diagnostic.cs

```diff
@@ -104,6 +104,35 @@ public abstract partial class Diagnostic : IEquatable<Diagnostic>, IFormattable
             IEnumerable<Location> additionalLocations,
             ImmutableDictionary<string, string> properties,
             params object[] messageArgs)
+        {
+            return Create(descriptor, location, effectiveSeverity: descriptor.DefaultSeverity, additionalLocations, properties, messageArgs: messageArgs);
```

Comment:
Why use named args for `messageArgs`?

Reply:
Ah, that came from copying the code above. Will fix it.